### ISSUE TITLE
Change Font and Remove Automatic TextTransform to Upper Case

### ISSUE THREAD

**Issue by pabhermoso** [PrathamJha1/st_tabs#1]
Hello! This is great. Would it be possible to adjust both the font and remove the Upper Case text transform?

Thanks.

**Comment by PrathamJha1** [comment]
Hey, that's a good point.We can add this control for the user but does not see the real implications.

**Comment by pabhermoso** [comment]
I am actually interested in getting it to look similar enough to the basic st.tabs version. So these changes are fpr visual purposes. Thanks!

**Comment by PrathamJha1** [comment]
Yup those changes can be added as well. I will look into it when i get the time. Feel free to suggest new things!

**Comment by nimeshrevo** [comment]
Is this solved, or any workaround found for it, especially the uppercase?

**Comment by PrathamJha1** [comment]
Currently not looking forward to fixing these changes
